### DISCUSSION TITLE
Rendering outside of index.js

### DISCUSSION THREAD

**Discussion by Pascapone** [jeffreylanters/react-unity-webgl#202] [Help Wanted!] [answered]
Hi there,
first of all i love your project! Sending and recieving data between unity and react works like a charm. Your documentation is amazing, thanks alot for the great work. I have a problem tho, maybe you can help. When i try to render my game compenent outside of index.js i get the following error message: "index.js:1451 A problem occurred while mounting Error: Unable to invoke Unity Loader, loaded incorrectly.  at UnityLoaderService.<anonymous>". Usually i got this message when my build wasnt in public or i got the name wrong. So it might be caused by a wrong path? But as soon as i move my component to index.js it works again. Also i have to mention that i use the react router.
Thank you for your time
Pasko

**Comment by Pascapone** [comment, marked as answer]
I solved it, just put / infront of the urls

const unityContext = new UnityContext({
  loaderUrl: "/WebGlBuild/build.loader.js",
  dataUrl: "/WebGlBuild/build.data.unityweb",
  frameworkUrl: "/WebGlBuild/build.framework.js.unityweb",
  codeUrl: "/WebGlBuild/build.wasm.unityweb",
});

**Reply by jeffreylanters** [reply]
Hii! Thank you very much. I'm glad you've solved it! There reason for this behaviour is because the file paths are relative to your application's server root rather than the working JavaScript file. Best of luck with your project! 🚀
